src/askui_runner/project_template/helper/jest.setup.ts.jinja

```diff
@@ -1,10 +1,24 @@
 import { UiControlClient } from 'askui';
 import { AskUIAllureStepReporter } from '@askui/askui-reporters';
+import {Blob} from 'node:buffer';
 
 let aui: UiControlClient;
 
 jest.setTimeout(60 * 1000 * 60);
 
+// https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html
+// Limit is set to 9.9 that also metadata can be transferred in request
```

Comment:
Would also mention which limit is expected (10) (optional from my side)

```suggestion
// Limit is set to 9.9 instead of 10 to account for metadata (not part of file / request body) that is also transferred with request
```